contrail.sh

```diff
@@ -76,7 +76,7 @@ LAUNCHPAD_REPO=${LAUNCHPAD_BRANCH:-snapshots}
 
 USE_DISCOVERY=${USE_DISCOVERY:-False}
 
-RABBIT_USER=${RABBIT_USER:-guest}
+RABBIT_USER=${RABBIT_USER:-rabbitcontrail}
```

Comment:
As log as , RABBIT_IP=127.0.0.1, 'guest' rabbit user will work.
But if changed SERVICE_HOST to physical interface IP, single node setup will not work.
On single node setup, I set SERVICE_HOST=192.168.1.15
which set RABBIT_IP=192.168.1.15, hence api-server failed to connect with 'guest' user.

This pull request will fix sinlge and multi node rabbitmq connection issue.
Also, devstack uses, 'stackrabbit' user instead of 'guest'